<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AuthPage</class>
    <widget class="QWidget" name="AuthPage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>640</width>
                <height>480</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Log in to your account</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <property name="alignment">
                <set>Qt::AlignHCenter|Qt::AlignCenter</set>
            </property>
            <property name="spacing">
                <number>10</number>
            </property>
            <property name="leftMargin">
                <number>120</number>
            </property>
            <property name="rightMargin">
                <number>120</number>
            </property>
            <item>
                <widget class="QLabel" name="titleLabel">
                    <property name="text">
                        <string>Log in</string>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>20</pointsize>
                            <bold>true</bold>
                            <weight>75</weight>
                        </font>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeType">
                        <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLineEdit" name="usernameInput">
                    <property name="placeholderText">
                        <string>Username</string>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">
                            border-radius: 5px;
                            border: 1px solid #ccc;
                            padding: 10px;
                            font-size: 14px;
                        </string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="passwordInput">
                    <property name="placeholderText">
                        <string>Password</string>
                    </property>
                    <property name="echoMode">
                        <enum>QLineEdit::Password</enum>
                    </property>
                    <property name="styleSheet">
                        <string notr="true">
                            border-radius: 5px;
                            border: 1px solid #ccc;
                            padding: 10px;
                            font-size: 14px;
                        </string>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout">
                    <item>
                        <widget class="QLabel" name="errorLabel">
                            <property name="text">
                                <string/>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>10</pointsize>
                                </font>
                            </property>
                            <property name="wordWrap">
                                <bool>true</bool>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">
                                    color: red;
                                </string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="loginButton">
                            <property name="text">
                                <string>Login</string>
                            </property>
                            <property name="font">
                                <font>
                                    <pointsize>12</pointsize>
                                    <bold>true</bold>
                                    <weight>75</weight>
                                </font>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">
                                    QPushButton {
                                        border-radius: 5px;
                                        background-color: #000;
                                        color: #fff;
                                        padding: 10px;
                                        font-size: 14px;
                                        max-width: 150px;
                                    }
                                    QPushButton:hover {
                                        background-color: #111;
                                    }
                                </string>
                            </property>
                            <property name="cursor">
                                <cursorShape>PointingHandCursor</cursorShape>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
